ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -165,6 +165,33 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# Transceiver Settings #}
+{%     set transceiver_settings = [] %}
+{%     for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
+{%         if ethernet_interface.transceiver is arista.avd.defined %}
+{%             do transceiver_settings.append(ethernet_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if transceiver_settings | length > 0 %}
+
+##### Transceiver Settings
+
+| Interface | Transceiver Frequency | Media Override |
+| --------- | --------------------- | -------------- |
+{%         for transceiver_setting in transceiver_settings | arista.avd.natural_sort('name') %}
+{%             set interface = transceiver_setting.name | arista.avd.default('-') %}
```

Comment:
```suggestion
{%             set interface = transceiver_setting.name %}
```
I dont think we need default as interface name. Since its primary key and in sort we are using name

Reply:
makes sense, fixed